core/workspace_comment/workspace_comment_render_svg.js

```diff
@@ -97,22 +93,44 @@ Blockly.WorkspaceCommentSvg.prototype.createEditor_ = function() {
   body.appendChild(textarea);
   this.textarea_ = textarea;
   this.foreignObject_.appendChild(body);
-  Blockly.bindEventWithChecks_(textarea, 'mouseup', this, this.textareaFocus_);
   // Don't zoom with mousewheel.
   Blockly.bindEventWithChecks_(textarea, 'wheel', this, function(e) {
     e.stopPropagation();
   });
   Blockly.bindEventWithChecks_(textarea, 'change', this, function(
       /* eslint-disable no-unused-vars */ e
       /* eslint-enable no-unused-vars */) {
-    if (this.text_ != textarea.value) {
+    if (this.content_ != textarea.value) {
       Blockly.Events.fire(new Blockly.Events.BlockChange(
-        this.block_, 'comment', null, this.text_, textarea.value));
-      this.text_ = textarea.value;
+        this.block_, 'comment', null, this.content_, textarea.value));
+      this.content_ = textarea.value;
     }
   });
   setTimeout(function() {
     textarea.focus();
   }, 0);
   return this.foreignObject_;
 };
+
+/**
+ * Returns this comment's text.
+ * @return {string} Comment text.
+ */
+Blockly.WorkspaceCommentSvg.prototype.getContent = function() {
```

Comment:
This and setContent belong in workspace_comment_svg, not workspace_comment_render_svg